### DISCUSSION TITLE
Send Later column comes back each time Thunderbird starts

### DISCUSSION THREAD

**Discussion by tugkanbatu** [Extended-Thunder/send-later#471] [Q&A] [answered]
I currently use Send Later 9.0.11 on Thunderbird 102.2.2 (on Ubuntu).

My nuisance is that each time I start Thunderbird, "Send Later" column is displayed by default in my Inbox messages pane. I have unchecked the options "Show Send Later Column" and "Show Send Later Header" (see the attached screenshot) in the preferences. Still, each time Thunderbird starts, the column is displayed even though the preferences in the extension's setting indicate otherwise. Actually, as soon as I go into the preferences of the extension, the column is removed. So, it seems like the preference is not taken into account until I view the preferences window for the extension. It is annoying I have to remove the column each time either by using the "select columns to display" icon or by going into the extension's preferences.

Any help would be appreciated.

Tugkan

![Screenshot from 2022-11-06 15-18-25](https://user-images.githubusercontent.com/24612118/200179559-2fa21b06-4586-4aec-a601-a0471b23fde7.png)

**Comment by jikamens** [comment, marked as answer]
I believe this is fixed in the current beta release (9.2.2),.
